src/main/cc/wfa/measurement/common/crypto/constants.h

```diff
@@ -56,6 +56,8 @@ inline const std::pair<std::string, std::string> kGenerateNewCompositeCipher = {
     "", ""};
 inline const std::pair<std::string, std::string>
     kGenerateNewParitialCompositeCipher = {"", ""};
+// Elliptic curve to be used when none is specified.
```

Comment:
We should indicate what curve this is (the name of the curve), right now it is just a magic number.